### PR TITLE
va11halla

### PR DESCRIPTION
**Pull request by fernandos33**
new reagents and machine for vall drinks, waiting for undudo sprites 
new machine the  CALICOMP
which will dispense the reagents/drinks
Adelhyde
Flanergide
Bronson_Extract
Powdered_Delta
Karmotrine

any doubts with the code ask me, anything else is with undudo (i|m only dealing with the code)
  (NOT READY FOR REVIEW; SPRITES STILL MISSING)

### CODE REVIEW

**Comment by seekin4u** [comment]
`code/modules/reagents/reagents/food-Drinks.dm:1007:error: missing left-hand argument to +.`

**Comment by seekin4u** [comment]
Pwease compile before pufhing OwO

**Comment by fernandos33** [comment]
READY FOR REVIEW

**Comment by fernandos33** [comment]
sprites are in too °°

**Comment by Kurgis** [comment]
Are most of these drink descriptions copy-pasted? They don't seem to make any sense, as they seem to all have a different context. One drink directly refers to the viewer's own observation of the moon (implied as the one above Earth), which wouldn't be possible for any character. Another description tells the viewer that this is exactly how "Jill" described it, these references are way too direct.

The sprites are very nice.  👍

**Comment by SpadesSlick** [comment]
We need screenshotsin description, for all the things you add with that PR .

**Comment by granodd** [comment]
What's the status on this? Screenshots would be really helpful, though if sprites aren't there I do remember spriting a dispenser that could possibly be used. Also if the drinks are put in, it would be better to just put them in the base dispenser as opposed to making a special one just for the additives.

**Comment by Kurgis** [comment]
@granodd https://va11halla.fandom.com/wiki/Drinktionary
You can view the sprites on each drink page.

They are quite out of place with the rest of the drink line-up in my opinion, they should be kept more separated from the rest of the drinks in the game.

**Comment by granodd** [comment]
So from what I could gather there, it's the combination of base chemicals to form alcoholic drinks, and actually make all of the drinks entirely. The concept is interesting though I'm unsure of how much it fits in with the current system. It would be better to have one system or the other, and I think it's better to keep what we have.

It may be better to create new chemicals or drinks that are inspired by Valhalla as opposed to being directly from it that can be mixed into existing drinks to either add effects to them, or to create newer drinks which can be added. Putting those chemicals or drinks in place for the future would be good, as I believe adding stat modification through drinks was planned, and so these chemicals could play a part in that.

Though I still do not think adding a new dispenser is a good idea. Either putting the chemicals into an existing dispenser, or putting them into their own bottles for manual mixing would be preferable in my eyes.

**Comment by ACCount12** [comment]
I'm not sure that the idea, or the implementation for that matter, fits into our server well.

**Comment by Kurgis** [comment]
Personally, I'd believe it'd be better to change the art style of existing drinks to fit the style of Va11-Halla as a reference to said game, instead of cropping a select few drinks from the game itself.

File: code/modules/reagents/recipes.dm [hunk 1]
```diff
@@ -2079,3 +2079,297 @@
...
+	result_amount = 2
+
+/datum/chemical_reaction/Deltabron
+	name = "Deltabron"
+	id = "Deltabron"
+	result = "Deltabron"
+	required_reagents = list("Powdered_Delta" = 1, "Bronson_Extract" = 1)
+	result_amount = 2
+
+	/datum/chemical_reaction/Deltaflan
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/recipes.dm]: Why is this one tabbed?

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 2]
```diff
@@ -1288,6 +1402,650 @@
...
+	reagent_state = LIQUID
+	color = "#79c7ce"
+	strength = 26
+
+	glass_icon_state = "Piano_Womanglass"
+	glass_name = "Piano Woman"
+	glass_desc = "It was originally called Pretty Woman, but too many people complained there should be a Piano Woman if there was a Piano Man."
+	glass_center_of_mass = list("x"=16, "y"=7)
+
+datum/reagent/ethanol/Pile_Driver
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/food-Drinks.dm]: I think a lot of these are missing /'s in front, or is this intended?

**Comment by drexample** [comment]
Wait, are the sprites ripped directly from a game? I'm pretty sure we can't use that.

**Comment by Kurgis** [comment]
@drexample Sprites are ripped directly, I was told that the images were cropped from the game itself. A better solution is to change the sprite style of existing drinks to reference VA11-HA11A, not taking a direct rip from a licensed title. There are differences from having noticeable design/style choices which similarly reflect another title (i.e. the floors and Starmancer), and cropping an other image from a game's files to put into a .dmi. I've never played VA11-HA11A, but had I originally known the way the sprites were taken, I'd of spoken this concern earlier. Even the descriptions are often done without a nuance to them, I believe this PR should be closed.

@fernandos33  has done work over here on Eris, and has been looking into our partner server Persistence Station. This is the largest project they've done so far, and if it gets closed it can be truly discouraging. I know when some of my projects have had push back, I simply stopped working. I only wish this was known earlier before this progress was made, I was under the assumption that it was spritework based upon the drinks, not literally the same spritework used in-game for them.

Unless VA11-HA11A has some weird kind of licensing agreement where this is okay, or if replacement sprites can be acquired, I don't see this PR moving forward from where it is.

**Comment by drexample** [comment]
@Kurgis Thank you for clarifying. Well, the source material is cross-platform licensed game (which also released on Nintendo's platform and those bastards sued for less), and a lot of PR's contents are unadapted raw material, ripped straight from the game or the wiki (like item descriptions mentioning in-game characters or in-universe events, not correlating with Eris lore.) This would be a violation of a license and also an addition of content, unsuitable for using in the game itself.

Creating some reference drinks, basing on already existing reagents is one thing and making a carbon copy of another game's content is the other. I can see that quite a bit of effort had been put in this PR, but this isn't something we can merge at any point. 

@fernandos33 I am sorry, but in 24 hours I will close this PR.

**Pull request closed without merging**